.github/workflows/promote.yaml

```
@@ -0,0 +1,79 @@
+            os: ubuntu-24.04-arm
+          - environment: dev
+            previous: library
+          - environment: test
+            previous: dev
+          - environment: prod
+            previous: test
+    runs-on: ${{ matrix.os }}
+    environment: ${{ matrix.environment }}
+    steps:
+      - name: Login to GHCR
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)